test/functional/test_framework/mininode.py

```diff
@@ -1604,7 +1604,10 @@ def wait_for_getheaders(self, timeout=60):
         assert wait_until(test_function, timeout=timeout)
 
     def wait_for_inv(self, expected_inv, timeout=60):
-        test_function = lambda: self.last_message.get("inv") and self.last_message["inv"] != expected_inv
+        """ Currently only checks the first inv object in an inv message"""
```

Comment:
In which case you might want to `raise NotImplementedError` if `len(expected_inv) >= 2`?